packages/ui-components/src/components/Dialog.tsx

```diff
@@ -59,7 +58,26 @@ const CloseButton = styled(IconButton)`
   color: ${props => props.theme.palette.text.primary};
 `;
 
-export const DialogHeader = ({ title, onClose, color, children }) => (
+interface DialogHeaderProps {
+  title: string;
+  onClose: () => void;
+  color?:
```

Comment:
Hopefully this can be imported from material-ui TypographyTypeMap? Otherwise we should create a shared type for it somewhere.